### ISSUE TITLE
Can't start the board!

### ISSUE THREAD

**Issue by Ruslan184** [scrapcomputing/MCEBlaster#11]
Greetings! I made a board, flashed Pico, but nothing happens when connected to a PC. I soldered another board, but the result is the same. Any ideas why I can't get the board to work?
![IMG_20241109_195922_356](https://github.com/user-attachments/assets/4309f7c6-6eb6-410b-b7db-09b9919cf100)
![IMG_20241109_200036_195](https://github.com/user-attachments/assets/ed807c6c-f99f-40c5-9f64-de9e2dfdaf2c)
![IMG_20241109_200056_227](https://github.com/user-attachments/assets/5e382b56-1d2f-4732-9618-6efb624a182c)

**Comment by scrapcomputing** [comment]
It could be that the flash chip isn't fast enough to handle the overclocked frequency. 
Could you try the attached firmware ?
[MCEBlaster_slowflash.uf2.zip](https://github.com/user-attachments/files/17689130/MCEBlaster_slowflash.uf2.zip)

**Comment by Ruslan184** [comment]
Everything works! Thank you very much!
![IMG_20241109_222726_700](https://github.com/user-attachments/assets/a04676e8-075c-439e-a6a2-893d8cdcb0bd)
![IMG_20241109_222755_056](https://github.com/user-attachments/assets/d5d89f48-53c4-4e82-91f3-5c627cf44bdd)
![IMG_20241109_222812_473](https://github.com/user-attachments/assets/637256ef-4dc2-4dc1-951b-c73c13c0e0c5)
With the new firmware, the LED lights up when connected.

**Comment by scrapcomputing** [comment]
That's great!
I have not tested this firmware much, so please let me know if you run into any issues. Thanks!

**Comment by Ruslan184** [comment]
Ні! Can't center the image. Cropped on the left. Is it possible to fix it somehow? Monitor 17"
![IMG_20241117_175112_819](https://github.com/user-attachments/assets/4d52a5fd-ad89-49b0-a04c-fd229fb380f1)

**Comment by scrapcomputing** [comment]
1. What mode is this, is it MDA, EGA 640x350 or EGA 640x200 ?
2. Does this happen in all modes or applications?
3. I assume you have tried centering the image using the monitor's settings, but just to make sure you have already tried it.
4. Could you try the attached firmware to see if it fixes the issue.
[MCEBlaster_rev0.1c_XB0.uf2.zip](https://github.com/user-attachments/files/17791554/MCEBlaster_rev0.1c_XB0.uf2.zip)

**Comment by Ruslan184** [comment]
EGA 640x200
With the new firmware, everything is the same.
I am trying to adjust the image on both the MCEBlaster and the monitor.
The situation is somewhat better on the Samsung SincMaster EX1920 monitor.

**Comment by scrapcomputing** [comment]
I am trying to understand whether this is an issue with the TTL input section or the VGA output section.
1. Does this happen in other programs that fill the screen with color? Some LCD monitors have issues detecting the borders when the screen is mostly black.
2. Please try this new firmware.
[MCEBlaster_XBs4.uf2.zip](https://github.com/user-attachments/files/17791743/MCEBlaster_XBs4.uf2.zip)

**Comment by Ruslan184** [comment]
Everything is ok in other programs.
Tried the new firmware - everything is great! Thank you very much!
I have 2 EGA cards and both works great.
![IMG_20241117_195134_031](https://github.com/user-attachments/assets/166450b2-3741-4ba6-8bca-082f8c5ab75a)
![IMG_20241117_195158_922](https://github.com/user-attachments/assets/d0e5346b-f2de-4f61-ba2d-bf96c4d27647)
![IMG_20241117_195252_113](https://github.com/user-attachments/assets/8520b4dc-0730-4f99-9c39-223d3d6107dc)
![IMG_20241117_195316_018](https://github.com/user-attachments/assets/61df6cfc-bb60-4bb7-a845-021a80b4433a)
![IMG_20241117_195454_477](https://github.com/user-attachments/assets/5faabe14-09b4-4d9f-a755-a43b522d48fc)
![IMG_20241117_195535_439](https://github.com/user-attachments/assets/4636cbd3-3441-4e56-b4fa-ac55fdec5eb8)
![IMG_20241117_195618_372](https://github.com/user-attachments/assets/97ad2a40-3b07-4e31-a192-809336ddd66e)
![IMG_20241117_195803_976](https://github.com/user-attachments/assets/d7aca647-f430-4945-a2ef-3002c888b2fe)

**Comment by scrapcomputing** [comment]
OK, so the issue is on the TTL side, the border detection is slightly off.
It does look better, but it seems that some pixels are still missing though. This firmware might be a little better:
[MCEBlaster_XBs8.uf2.zip](https://github.com/user-attachments/files/17792038/MCEBlaster_XBs8.uf2.zip)

**Comment by Ruslan184** [comment]
The latest firmware cuts the image already on the right.
The ideal would be something in between XBs4 and XBs8.
![IMG_20241118_203221_470](https://github.com/user-attachments/assets/db68e7f2-e7d1-4353-b956-4af228ee6c93)
![IMG_20241118_203331_985](https://github.com/user-attachments/assets/086be3ac-6106-4977-9757-f5df1dba0dbb)
![IMG_20241118_203424_576](https://github.com/user-attachments/assets/52965dfc-3d7b-4cf5-b032-d9b705ec5f80)

**Comment by scrapcomputing** [comment]
Thanks for trying it out, I will see what I can do.
